flight/Libraries/math/pid.c

```diff
@@ -130,10 +130,22 @@ float pid_apply_antiwindup(struct pid *pid, const float err,
  * This version of apply uses setpoint weighting for the derivative component so the gain
  * on the gyro derivative can be different than the gain on the setpoint derivative
  */
-float pid_apply_setpoint(struct pid *pid, const float setpoint, const float measured, float dT)
+float pid_apply_setpoint(struct pid *pid, struct pid_deadband *deadband, const float setpoint,
+	const float measured, float dT)
 {
 	float err = setpoint - measured;
-	
+	float err_d = (deriv_gamma * setpoint - measured);
+
+#ifndef SMALLF1
```

Comment:
This ifdef shouldn't be here-- it doesn't save RAM on F1-- just a small amount of code space-- but silently doesn't apply any deadband that you *have* allocated.